code/src/reducers/game.js

```diff
@@ -0,0 +1,69 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
```

Comment:
Good with an own variable for the initialstate, makes it easier to read and follow